applications/CompressiblePotentialFlowApplication/python_scripts/compute_lift_process.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, Model, settings ):
         self.create_output_file = settings["create_output_file"].GetBool()
 
     def ExecuteFinalizeSolutionStep(self):
-        print('COMPUTE LIFT')
+        KratosMultiphysics.Logger.PrintInfo('ComputeLiftProcess3D','COMPUTE LIFT')
```

Comment:
```suggestion
        KratosMultiphysics.Logger.PrintInfo('ComputeLiftProcess','COMPUTE LIFT')
```
In fact we are using this for 2D as well! I would remove the dimension for now

Reply:
Yes that's true! I'll fix it now.